### PR TITLE
Add licensing information

### PR DESCRIPTION
**Pull request by open-source-explorer**
Add licensing information to the `README` file which is a usual practice being followed 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,11 @@ This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) ma
 
 We are currently only accepting bug fixes for Gatsby v1. No new features will be accepted.
 
+### License
```
Comment (leachjustin18) [inline README.md]: To follow the style guide as the rest of the read me:
```suggestion
## License
```
Adding an icon next to License.

File: README.md [hunk 2]
```diff
@@ -153,6 +153,11 @@ This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) ma
...
+
+
```
Comment (lipis) [inline README.md]:
```suggestion
```

**Comment by leachjustin18** [comment]
Hi @open-source-explorer,

Thank you for addressing the header comment.  It may have been lost, in the comment above, I asked if we can add an icon next to License to also follow the style?

**Comment by open-source-explorer** [comment]
@leachjustin18 Does this one makes sense to you? 📝

File: README.md [hunk 3]
```diff
@@ -153,6 +153,10 @@ This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) ma
 
 We are currently only accepting bug fixes for Gatsby v1. No new features will be accepted.
 
+## :memo: License
+
+ Licensed under the [MIT License](https://github.com/gatsbyjs/gatsby/blob/master/LICENSE)
```
Comment (lipis) [inline README.md]:
```suggestion
Licensed under the [MIT License](https://github.com/gatsbyjs/gatsby/blob/master/LICENSE)
```

To fix the linting error.

**Comment by open-source-explorer** [comment]
@lipis @leachjustin18 Is it fine now?

**Pull request merged**